plugin/package.json

```diff
@@ -106,6 +106,7 @@
   "dependencies": {
     "easymde": "2.18.0",
     "immutability-helper": "3.1.1",
+    "js-file-download": "^0.4.12",
```

Comment:
Remember to add deps with specific versions

Reply:
Done!